scipy/sparse/_compressed.py

```diff
@@ -1407,6 +1407,48 @@ def _divide_sparse(self, other):
             return out
 
 
+    def broadcast_to(self, shape):
+        if self.format != 'csr':
```

Comment:
Let's move this method to the CSR class directly, which will avoid the need to check format here.

Reply:
Moved to class `_csr_base` - fd933de72b8289902921c5b5606e1ad980c8751e